blueprints/load-scheduling/jmx/config.libsonnet

```diff
@@ -0,0 +1,28 @@
+local serviceProtectionDefaults = import '../common/config-defaults.libsonnet';
+
+
+serviceProtectionDefaults {
+  policy+: {
+    /**
+    * @param (policy.promql_query: string) PromQL query.
+    */
+    promql_query: 'avg(java_lang_G1_Young_Generation_LastGcInfo_duration{k8s_pod_name=~"service3-demo-app-.*"})',
```

Comment:
This is a hardcoded query! No need to expose the query as we discussed. The blueprint needs to be specific for garbage collection. Filter by the infra meter name and policy name in the query